packages/apps/todomvc/src/main.css

```diff
@@ -1,7 +1,36 @@
-button#open {
-  position: absolute;
-  top: 15px;
-  left: 15px;
+div#spaces button {
```

Comment:
is the thinking that adding vite/tailwind is too much?

Reply:
just avoiding in todomvc because 99% of the styles are already provided and it's just a simple example